schema/application/checklist.schema.graphql

```diff
@@ -30,6 +30,11 @@ type Query {
 type Mutation {
   copyFrom(entity: ID!, options: CopyFromOptions!): CopyFromPayload!
   saveChecklist(input: ChecklistInput!): ChecklistEdge!
+  discardChecklist(entity: ID!): DiscardChecklistPayload!
+}
+
+type DiscardChecklistPayload {
+  edge: ChecklistEdge!
 }
```

Comment:
This should return the discarded checklist (ids) as well, so we can use the `@deleteEdge` directive (or manually via `ConnectionHandler.deleteEdge`)

Something like;
```graphql
type DiscardChecklistPayload {
  discardedChecklistIds: [ID!]!
  edge: ChecklistEdge!
}
```

in client code we can then do
```graphql
mutation DiscardChecklist($entity: ID!, $connections: [ID!]!) {
  discardChecklist(entity: $entity) {
    discardedChecklistIds @deleteEdge(connections: $connections)
    edge @preprendEdge(connections: $connections)
  }
}
```

(or similar using the manual approach via `ConnectionHandler`)

https://relay.dev/docs/guided-tour/list-data/updating-connections/#deleteedge